internal/hud/model/model.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"time"
+
+	"github.com/windmilleng/tilt/internal/model"
+)
+
+type Pod struct {
+	Name      string
+	StartedAt time.Time
+	Status    string
+}
+
+type Resource struct {
+	DirectoryWatched   string
+	LatestFileChanges  []string
+	LastFileChangeTime time.Time
+	Status             ResourceStatus
+}
+
+type ResourceStatus int
+
+const (
+	ResourceStatusUnbuilt ResourceStatus = iota
+	ResourceStatusStale
+	ResourceStatusFresh
+)
+
+func NewResource(directoryWatched string) Resource {
+	return Resource{
+		DirectoryWatched:   directoryWatched,
+		LatestFileChanges:  []string{},
+		LastFileChangeTime: time.Now(),
+		Status:             ResourceStatusUnbuilt,
+	}
+}
+
+type Model struct {
```

Comment:
this is a pretty un-descriptive name. if you want to call it `model` i'd put in the `hud` package so it'd descriptively/usefully imported as `hud.Model`; if you want to leave it here, maybe call it something more descriptive?

Reply:
actually, wait, why does this need its own package at all? why not `internal/hud/model.go`?

Reply:
I had model and view in the same package, but then `model.ResourceStatus` conflicts with `view.ResourceStatus`, and having `model.ResourceStatus` as a peer to `view.ResourceStatus` seems nicer? 

`model` is super annoying since it conflicts with `internal/model`, and I had to go through and do renaming imports everywhere, which is probably a good reason to do something different.

Moving this into `engine` seems like a good solution.